jakarta/src/main/java/com/fasterxml/jackson/module/jsonSchema_jakarta/annotation/JsonHyperSchema.java

```diff
@@ -0,0 +1,95 @@
+package com.fasterxml.jackson.module.jsonSchema_jakarta.annotation;
```

Comment:
this is very non-standard for a package name - normally only lowercase a-z, 0-9 and dot separators.

Reply:
could you change the underscore to a '.'?

Reply:
Agreed about the lowercase, but `jsonShema` part is just a copy of the pre-existing module .
IME, underscore are conventionals.

Reply:
> could you change the underscore to a '.'?
IMO, doing that will lead to JPMS split package issue.